vite.config.ts

```diff
@@ -31,5 +31,6 @@ module.exports = defineConfig({
       formats,
       fileName: (format) => fileName[format],
     },
+    emptyOutDir: false
```

Comment:
Since we `tsc` before we build with `vite`, we don't want `vite` to wipe out the declarations file created by `tsc`.